JellyfinPlayer/SettingsView.swift

```diff
@@ -25,101 +27,100 @@ struct SettingsView: View {
     @Default(.videoPlayerJumpBackward) var jumpBackwardLength
 
     var body: some View {
-        NavigationView {
-            Form {
-                Section(header: EmptyView()) {
+        Form {
+            Section {
+                HStack {
+                    Text("User")
+                    Spacer()
+                    Text(SessionManager.current.user.username ?? "")
+                        .foregroundColor(.jellyfinPurple)
+                }
+
+                Button {
+                    settingsRouter.route(to: .serverDetail)
+                } label: {
                     HStack {
-                        Text("User")
+                        Text("Server")
                         Spacer()
-                        Text(SessionManager.current.user.username ?? "")
+                        Text(ServerEnvironment.current.server.name ?? "")
                             .foregroundColor(.jellyfinPurple)
```

Comment:
The removal of the NavigationLink removes the right chevron on the row for this. Let's add one in here to indicate that it will navigate to a new view.